doc/fluid/design/dist_train/prefetch_parameter.md

```diff
@@ -0,0 +1,48 @@
+# Design Doc: Prefetching Parameter From Parameter Server
+
+## Abstract
+
+We propose an approach to prefetch parameter from Parameter
```

Comment:
_pre-fetch_

Reply:
It should be:
We propose an approach to **pre-fetch** **the** parameters from **a** Parameter Server while distributed training so that Fluid **is able to train a model with a large number of parameters** that cannot be stored in one trainer's memory.